css/index.css

```diff
@@ -18,6 +18,8 @@ header{
           
 }
 
+/* Navigaition-------------------------------------------------------------------*/
```

Comment:
There's a typo here.

```suggestion
/* Navigation-------------------------------------------------------------------*/
```